src/components/KDropdown/KDropdownItem.vue

```diff
@@ -0,0 +1,303 @@
+<template>
```

Comment:
Can you point out any changes in this file? I can't view the diff since it's a new file.

Reply:
Yes: https://github.com/Kong/kongponents/compare/alpha...104defa39f2520e795deab8a1cc6096762e8a3b5
This is a diff comparing last commit before file rename to base (`alpha`). Changes to template and script after this commit.